agent/agent_test.go

```diff
@@ -63,15 +66,20 @@ func TestAgentSuite(t *testing.T) {
 	suite.Run(t, new(AgentSuite))
 }
 
+// kim: TODO: fix merge conflict due to task working directory consolidation
 func (s *AgentSuite) SetupTest() {
 	var err error
+	s.tmpDirName, err = os.MkdirTemp("", filepath.Base(s.T().Name()))
```

Comment:
Nit: use `s.T().TempDir()`

Reply:
This isn't relevant anymore after merging #7030.